### ISSUE TITLE
When the killer dies, resurrect the player

### ISSUE THREAD

**Issue by doorz1009** [doorz1009/teeworlds#6]
Resurrect victim when the killer dies.

**Comment by doorz1009** [comment]
This issue addressed the requirements:
_5. Players shall not respawn until their killer is killed.
6. All victims on a player's list shall respawn upon the player's death._

**Comment by doorz1009** [comment]
This issue is dependent upon Issue #5.

**Comment by doorz1009** [comment]
This has been implemented and merged into the master branch.
